cms/models.py

```diff
@@ -193,7 +194,7 @@ class CoursePage(ProductPage):
     Detail page for courses
     """
 
-    parent_page_types = [Page]
+    parent_page_types = [HomePage]
```

Comment:
@umarmughal824 By making it child of HomePage, user be not able to associate `ProductDetails` while creating the `HomePage`

Reply:
I mean, If there is not any product page exist then he cannot create it until not complete the HomePage creation and HomePage will not be created if not any Product is associated.

Reply:
the course pages are not working by making it a direct child of Page. You could see the attached screenshots of working attached products with the home page.

Reply:
@asadiqbal08 @umarmughal824 Please review this PR: https://github.com/mitodl/mitxonline/pull/81. I created this PR because we were going to create this index page anyway, but also because making `CoursePage` a child of `Page` instead of `HomePage` broke those links as Umar described. Merging that PR will fix this issue (at least that's what I observed)

Reply:
yeah exactly it breaks the link and neither it loads any course page nor it shows any preview or live button on the wagtail page. I could still attach those course pages to the home page by making it the child of the home page.